web_chatter_position/static/description/index.html

```diff
@@ -0,0 +1,138 @@
+                    <img class="img img-responsive center-block" src="images/pos_discount_restrict.png" style="border-top-left-radius: 10px;border-top-right-radius: 10px;"/>
+                  </div>
+                </a>
+            </div>
+            <div class="col-xs-6 col-sm-4 col-md-4 mb16 mt16" style="float: left;">
+                <a target="_blank" href="https://apps.odoo.com/apps/modules/15.0/ob_pos_payment_restriction/">
+                  <div style="box-shadow: 0 15px 35px rgba(50, 50, 93, 0.1), 0 5px 15px rgba(0, 0, 0, 0.07);border-radius: 10px;">
+                    <img class="img img-responsive center-block" src="images/pos_payment_restriction.png" style="border-top-left-radius: 10px;border-top-right-radius: 10px;"/>
+                  </div>
+                </a>
+            </div>
+            <div class="col-xs-6 col-sm-4 col-md-4 mb16 mt16" style="float: left;">
+                <a target="_blank" href="https://apps.odoo.com/apps/modules/15.0/ob_pos_global_discount_selection/">
+                  <div style="box-shadow: 0 15px 35px rgba(50, 50, 93, 0.1), 0 5px 15px rgba(0, 0, 0, 0.07);border-radius: 10px;">
+                    <img class="img img-responsive center-block" src="images/pos_global_discount_amount_percentage.png" style="border-top-left-radius: 10px;border-top-right-radius: 10px;"/>
+                  </div>
+                </a>
+            </div>
+            <div class="col-xs-6 col-sm-4 col-md-4 mb16 mt16" style="float: left;">
+                <a target="_blank" href="https://apps.odoo.com/apps/modules/15.0/ob_pos_tax_display/">
+                  <div style="box-shadow: 0 15px 35px rgba(50, 50, 93, 0.1), 0 5px 15px rgba(0, 0, 0, 0.07);border-radius: 10px;">
+                    <img class="img img-responsive center-block" src="images/pos_product_tax_display.png" style="border-top-left-radius: 10px;border-top-right-radius: 10px;"/>
+                  </div>
+                </a>
+            </div>
+        </div>
+        <!-- <div class="carousel-item" style="min-height: 0px;"> -->
+        <!-- </div> -->
+      </div>
+      <a class="carousel-control-prev" href="#demo" data-slide="prev" style="left:-25px;width: 35px;color: #000;"> <span class="carousel-control-prev-icon"><i class="fa fa-chevron-left" style="font-size:24px"></i></span> </a> <a class="carousel-control-next" href="#demo" data-slide="next" style="right:-25px;width: 35px;color: #000;"> <span class="carousel-control-next-icon"><i class="fa fa-chevron-right" style="font-size:24px"></i></span> </a>
+    </div>
+</section>
+
+<h3 class="oe_slogan" style="margin-top:20px;" >Need Any Help Or Have Any Feature Requests?</h3>
+<div class="oe_slogan" style="margin-top:10px !important;">
+    <div>
+        <a  class="btn btn-primary btn-lg mt8"
+        style="color: #FFFFFF !important;border-radius: 0;" href="mailto:hynsystechnologies@gmail.comm"><i
+        class="fa fa-envelope"></i> Email </a>
+    </div>
+    <br>
+</div>
+

```

Comment:
Since the module is maintained by OCA, the index and the images can be removed (or changed) I guess

Reply:
This file is going to be re-generated. You can delete it.